var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -51,7 +51,7 @@ class Warpx(MakefilePackage):
     depends_on('openpmd-api +mpi', when='+openpmd +mpi')
     depends_on('ascent', when='+ascent')
     depends_on('ascent +cuda', when='+ascent backend=cuda')
-    depends_on('ascent +mpi ^conduit~hdf5', when='+ascent +mpi')
+    depends_on('ascent +mpi', when='+ascent +mpi')
 
```

Comment:
This will now build conduit with hdf5 suport, which is good (I think folks will want to be able to save out Blueprint HDF5 files)

But i suspect that was there for a reason, conflict with non-mpi enabled and mpi-enabled hdf5s in the spec?

Reply:
yes, I also like to use *parallel* HDF5 with openPMD-api as an option :) I think the *serial* HDF5 constrain in conduit will conflict with that.

Reply:
you people and your fancy HDF5s :-)

For what its worth - the conduit spack package now longer imposes hdf5~mpi.  So I think we are ok, we don't need to properly incorp the sigil logic via other deps.